CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# Contributing
+
+
+
+## Contributing Time-Series Features
+
+We gratefully accept contributions of new time-series features, be they
+domain-specific or general. Please follow the below guidelines in order that
+your features may be successfully incorporated into the Cesium feature base.
```

Comment:
@stefanv bnaul offered to fill this out - he's going to contribute to my branch.